lib/utils/request.js

```diff
@@ -42,8 +42,13 @@ function createErrorMessage(err, httpResponse, body){
     err.requestId = requestId;
     return err;
   } else {
-    var msg = util.format(i18n._('Error - not 2xx status code. (%s)\n'), statusCode);
-
+    var errorMsgTxt = '';
+    if(statusCode === 401 || statusCode === 403){
+      errorMsgTxt = 'Authentication Error - run `fhc login` and try again';
+    } else {
+      errorMsgTxt = 'Error - not 2xx status code.';
+    }
+    var msg = util.format(i18n._( errorMsgTxt + '(%s)\n'), statusCode);
```

Comment:
Is server response displayed in this case? It usually looks really ugly so it will be better if we can suspend it.